apps/aehttp/src/aehttp_app.erl

```diff
@@ -15,11 +15,11 @@
 
 -define(DEFAULT_WEBSOCKET_INTERNAL_PORT, 8144).
 -define(DEFAULT_WEBSOCKET_LISTEN_ADDRESS, <<"127.0.0.1">>).
--define(INT_ACCEPTORS_POOLSIZE, 10).
+-define(INT_ACCEPTORS_POOLSIZE, 100).
```

Comment:
This should be renamed to `MAX_CONNECTIONS` instead, since the acceptor pool size is this number divided by ten:
```suggestion
-define(MAX_CONNECTIONS, 100).
```